Dockerfile

```diff
@@ -84,21 +107,27 @@ RUN mkdir -p ${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway && \
     install -D $(find ./third_party/googleapis/google/rpc -name '*.proto') -t /out/usr/include/google/rpc
 
 
-
 FROM alpine:${ALPINE_VERSION} as packer
 RUN apk add --no-cache curl
 
 ARG UPX_VERSION
 RUN mkdir -p /upx && curl -sSL https://github.com/upx/upx/releases/download/v${UPX_VERSION}/upx-${UPX_VERSION}-amd64_linux.tar.xz | tar xJ --strip 1 -C /upx && \
     install -D /upx/upx /usr/local/bin/upx
 
+# Use all output including headers and protoc from protoc_builder
 COPY --from=protoc_builder /out/ /out/
+# Use protoc and plugin from protoc_cs_builder
+COPY --from=protoc_cs_builder /out/usr/bin/protoc-3.11.2.0 /out/usr/bin/protoc-csharp
```

Comment:
where does `3.11.2` number come from?

Reply:
This is the build version of the protobuf included as submodule in `/third_party/protobuf`. So we don't have it in any variable. But I will try to use `protoc-*` for the copying.

I also will try to use the `-DgRPC_BUILD_GRPC_XXX_PLUGIN=OFF` flags to speed up the cmake build to only generate the C# plugin. Might speed it up a couple more minutes.